easybuild/easyconfigs/i/Inspector/Inspector-2017_update1.eb

```diff
@@ -13,4 +13,5 @@ dontcreateinstalldir = 'True'
 
 license_file = HOME + '/licenses/intel/license.lic'
 
+modextrapaths = {'PATH' : ['inspector/bin64'] }
```

Comment:
Yea, I think that would be ideal, then we can remove this from the modextrapaths from easyconfig